### PR TITLE
Start labeling files again

### PR DESCRIPTION
**Pull request by damccorm**
I updated our tag-ur-it bot to be a github action - https://github.com/damccorm/tag-ur-it. This should get it back up and running again.

To see it in action see my test repo here - https://github.com/damccorm/tasks-copy/blob/master/.github/workflows/label.yml - feel free to create issues there to try it out, it follows the same rules as this repo (except everything gets assigned to me).

### CODE REVIEW

**Comment by damccorm** [comment]
We should also update [issue-rules.yml](https://github.com/microsoft/azure-pipelines-tasks/blob/master/issue-rules.yml) to route things to the correct people. I'll add a best effort attempt to do that now.

**Comment by damccorm** [comment]
@Lovakumar you should be removed here right? Who should replace you?

**Comment by Lovakumar** [comment]
> @Lovakumar you should be removed here right? Who should replace you?

I will let you know. For now, keep my name.

File: .github/workflows/blank.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  label:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - uses: damccorm/tag-ur-it@master
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
+        configuration-path: "./issue-rules.yml"
```
Comment (jtpetty) [inline .github/workflows/blank.yml]: any way this could read the code owners file?

Comment (damccorm) [reply .github/workflows/blank.yml]: I think they're actually 2 seperate concepts. Codeowners assigns people based on area paths touched, this assigns people based on issue content. So while I think _maybe_ a mapping could make sense, its not immediately clear to me what that would look like or that it makes sense in the general case (I'd like to keep the action extensible if possible)

I think I'd probably prefer to take that as a potential future enhancement if that's ok.

Comment (jtpetty) [reply .github/workflows/blank.yml]: gotcha, sounds good

**Pull request merged**